README.md

```diff
@@ -34,29 +35,43 @@ Edit this document to include your answers after each question. Make sure to lea
 
 1. If you were to describe semantic HTML to the next cohort of students, what would you say?
 
+Semantic HTML consists of HTML tags and markups that give the content on the page meaning. An example of a NON semantic tag might be a <div>, which doesn't give the browser any information about the contents of the tag. Semantic HTML on the other hand would be something like <header>, <footer>, <nav>, <table>, etc. All of these tags queue us in on what and where the content might be. 
+
 2. Name two big differences between ```display: block;``` and ```display: inline;```.
 
+-Display: inline is unable to have its height and width modified while display: block can.
+-Display: inline takes up only the space needed to fit, display: block takes up the full width available and creates space above and below.
+
 3. What are the 4 areas of the box model?
 
+-Content
+-Padding
+-Border
+-Margin
+
 4. While using flexbox, what axis does the following property work on: ```align-items: center```?
 
+This works on the vertical axis (Y)
```

Comment:
Be careful with this, because the axis changes with the use of flex-direction: row vs. column.